src/pages/import/helpers.js

```diff
@@ -50,23 +73,29 @@ export function emitLogOnFirstResponse(xhr, importType) {
     }
 
     if (typeof response !== 'undefined') {
-        eventEmitter.emit('log', {
+        const emitLogData = {
             id: new Date().getTime(),
             d: new Date(response.created),
             type: `INFO - ${CATEGORY_2_LABEL[importType]}`,
             text: message,
-        })
+        }
+
+        debugFirstResponse(`Emit log, data: %O`, emitLogData)
```

Comment:
More template literals that should be a string literal, seems to be a trend so won't comment individually on them.